src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/CompletableFutures.java

```diff
@@ -0,0 +1,45 @@
+package io.openliberty.tools.intellij.lsp4mp.lsp4ij;
```

Comment:
Please keep the original header

Reply:
Oops, sorry I missed that. Fixed in new commit, along with LSPDiagnosticsForServer

Reply:
No problem, thanks for having fixed that.